src/Form/Control/Multiline.php

```diff
@@ -679,17 +679,15 @@ protected function renderView(): void
             throw new Exception('Multiline field needs to have it\'s model setup.');
         }
 
-        if ($this->cb->triggered()) {
-            $this->cb->set(function () {
-                try {
-                    return $this->renderCallback();
-                } catch (\atk4\Core\Exception $e) {
-                    $this->app->terminateJson(['success' => false, 'error' => $e->getMessage()]);
-                } catch (\Error $e) {
-                    $this->app->terminateJson(['success' => false, 'error' => $e->getMessage()]);
-                }
-            });
-        }
+        $this->cb->set(function () {
+            try {
+                return $this->renderCallback();
+            } catch (\atk4\Core\Exception $e) {
```

Comment:
catch in one block https://stackoverflow.com/questions/8439581/catching-multiple-exception-types-in-one-catch-block/37522012